File: 백준/Bronze/1075. 나누기/나누기.py
import sys

input = sys.stdin.readline
write = sys.stdout.write

def solve():
    n = int(input().rstrip())
    s = int(input().rstrip())
    if not n or not s: return
    
    target = (n // 100) * 100
    while target % s != 0: target += 1
    
    result = target % 100
    
    write(f"{result:02d}\n")
    
if __name__ == "__main__":
    solve()

File: 백준/Bronze/1075. 나누기/나누기.c
#include <stdio.h>

int main(void) {
	int n = 0, s = 0;
	if (scanf("%d %d", &n, &s) != 2) return 0;

	int target = (n / 100) * 100;
	while (target % s != 0) target++;

	int result = target % 100;

	printf("%02d\n", result);

	return 0;
}